openvdb/points/PointMove.h

```diff
@@ -510,11 +510,22 @@ struct GlobalMovePointsOp
     {
         auto& targetArray = mTargetHandles.getArray(targetOffset);
 
+// disable deprecated warnings for untyped copying of attribute values using ABI=6+
+// TODO: switch to using AttributeArray::copyValues() to migrate away from this work-around
+#if OPENVDB_ABI_VERSION_NUMBER >= 6
+#pragma GCC diagnostic push
```

Comment:
These generate "unrecognized pragma" warnings in ICC builds, and possibly with other compilers (VC++?).  See PR #309 for a more comprehensive fix.